src/hooks/http.js

```diff
@@ -0,0 +1,30 @@
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+
+const useHttp = (url, action, dependencies) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    fetch(url, {
```

Comment:
We shouldn't be performing fetching in our components/containers. API fetching shouldn't be in hooks but rather in their own services/utilities files